scripts/run_integration_tests_x86_64.sh

```diff
@@ -174,14 +174,14 @@ ulimit -l unlimited
 ulimit -n 4096
 
 export RUST_BACKTRACE=1
-time cargo test --release --target "$BUILD_TARGET" $test_features "common_parallel::$test_filter" -- ${test_binary_args[*]}
+time cargo test --features mshv,ivshmem --release --target "$BUILD_TARGET" $test_features "common_parallel::$test_filter" -- ${test_binary_args[*]}
```

Comment:
Please leave it unchanged and use a separate build to run ivshmem tests. The way how we test `dbus_api` below is a good reference.